sdks/python/tests/e2e/test_tracing.py

```diff
@@ -910,3 +910,61 @@ def f():
         attachments=attachments,
         data_sizes=data_sizes,
     )
+
+
+def test_track__function_called_with_mutable_input_which_changed_afterward__check_span_and_trace_inputs_are_not_affected(
```

Comment:
Let's not create an e2e test for this functionality as we're not testing anything related to the backend, I suggest creating a unit one.
For example, like in `test_tracker_outputs`. The test itself will remain almost the same, except for the part that we'll use fake backend and TraceModel/SpanModel classes instead of verifiers by id to check the final result.